dask/blockwise.py

```diff
@@ -319,6 +351,33 @@ def __dask_distributed_pack__(
         # All blockwise tasks will depend on the futures in `indices`
         global_dependencies = {stringify(f.key) for f in indices_unpacked_futures}
 
+        # Check if we need to serialize the elements of `io_deps`
+        # with `pickle` (i.e. something in the required inputs is
+        # not msgpack serializable)
+        if msgpack:
```

Comment:
is it even possible that this method gets invoked while msgpack isn't installed?

Reply:
I believe this was to fix an earlier CI failure where `msgpack` was not installed. Perhaps that was a temporary problem?